consumerBuilder.go

```diff
@@ -51,6 +51,11 @@ type (
 		saramaConfig        *sarama.Config
 		saramaClusterConfig *cluster.Config
 	}
+
+	consumerCluster struct {
```

Comment:
Can we add more comments, e.g. `name` here is cluster name instead of group name